README.md

```diff
@@ -80,6 +80,19 @@ Contributions of integrations with additional traffic sources are more than welc
 4. Open APIClarity UI in the browser: <http://localhost:9999/>
 5. Generate some traffic in the traced applications and check the APIClarity UI :)
 
+### Uninstall APIClarity from a K8s cluster using Helm:
+
+1. Helm uninstall
+   ```shell
+   helm delete apiclarity -n apiclarity
```

Comment:
`helm uninstall...`

Reply:
It's actually just an alias but we can change to uninstall, np.